### PR TITLE
Updating the script for the Antioch network

### PR DESCRIPTION
**Pull request by SurPaul**
https://discord.com/channels/811216481340751934/812344657794301972/842346670049591296

### CODE REVIEW

File: community-contributions/scriptnodeSetup/joystream-node-install.sh [hunk 1]
```diff
@@ -1,53 +1,42 @@
...
+[Install]
+WantedBy=multi-user.target" > joystream-node.service
+
+sudo mv joystream-node.service /etc/systemd/system/joystream-node.service
 sudo systemctl daemon-reload
 sudo systemctl start joystream-node
 sudo systemctl enable joystream-node
-clear
+
 echo -e "${GREEN}===============\nYour node is fully installed and running.\nNow go to the site (https://telemetry.polkadot.io/#list/Joystream) and find your node by the name you gave it.\n\nWait for complete synchronization and proceed to the next step - Validator Setup\n(read - https://github.com/Joystream/helpdesk/tree/master/roles/validators#validator-setup).\n===============\033[0m"
```
Comment (andybut) [inline community-contributions/scriptnodeSetup/joystream-node-install.sh]: You have to change the telemetry to https://telemetry.joystream.org/#/Joystream

File: community-contributions/scriptnodeSetup/joystream-node-install.sh [hunk 2]
```diff
@@ -1,53 +1,42 @@
...
-clear
+# My public key 5EyRepuyZTM2rjHwbGfjBmjutaR5QTxLULcra2aToigtpxb6
+# My node name wasabi
+
 echo "****************************************************************************"
 echo "*      This script will install and configure your Joystream node.         *"
 echo "*      The script was created based on the official instructions and the   *"
 echo "*      working community instructions:                                     *"
 echo "*      https://github.com/Joystream/helpdesk/tree/master/roles/validators  *"
 echo "*      https://seainvestments1.medium.com/b3608a8be10e                     *"
```
Comment (andybut) [inline community-contributions/scriptnodeSetup/joystream-node-install.sh:10]: this guide is not the best. I would remove the link to it. official enough.

**Review by bwhm — APPROVED** [review]
Looks good!

**Comment by bwhm** [comment]
Also thanks to @andybut for review :)

**Pull request merged**